src/libm_helper.rs

```diff
@@ -0,0 +1,143 @@
+use core::marker::PhantomData;
+
+use crate::*;
+
+pub struct Libm<T>(PhantomData<T>);
+
+macro_rules! libm_helper {
+	($t:ident, $(($libm_fn:ident => $func:ident($($arg:ident: $arg_type:ty),*) -> ($($ret_type:ty),*)));*) => {
+		impl Libm<$t> {
+
+			#![allow(unused_parens)]
+
+			$(
+				#[inline(always)]
+				pub fn $func($($arg: $arg_type),*) -> ($($ret_type),*) {
+					$libm_fn($($arg),*)
+				}
+			)*
+		}
+	};
```

Comment:
Can you fix the formatting here? Use spaces instead of tabs, and remove the blank lines.